packages/server/src/index.ts

```diff
@@ -62,9 +58,9 @@ export interface ServerAdapterObject<
   ): Promise<Response> | Response;
 }
 
-export type ServerAdapter<TServerContext, TBaseObject extends ServerAdapterBaseObject<TServerContext>> = TBaseObject &
-  ServerAdapterObject<TServerContext, TBaseObject>['handle'] &
-  ServerAdapterObject<TServerContext, TBaseObject>;
+export type ServerAdapter<TServerContext> = ServerAdapterObject<TServerContext>['handle'] &
+  ServerAdapterObject<TServerContext>['requestListener'] &
```

Comment:
Do we need `requestListener` here because `handle` already accepts `NodeRequest` and `ServerResponse` , no?

Reply:
You're right, I dropped it: https://github.com/ardatan/whatwg-node/pull/146/commits/5c1d634a333dde7cd021aeca190696ddd335974c.